src/lib/holocene/icon/svg/external-link.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import Svg from '../svg.svelte';
+</script>
+
+<Svg {...$$props}>
+  <g clip-path="url(#box)">
+    <path
+      d="M14.25 4H13.5V5.5H14.25H17.4406L10.4688 12.4688L9.9375 13L11 14.0594L11.5312 13.5281L18.5 6.55938V9.75V10.5H20V9.75V4.75V4H19.25H14.25ZM4.75 5H4V5.75V19.25V20H4.75H18.25H19V19.25V13.75V13H17.5V13.75V18.5H5.5V6.5H10.25H11V5H10.25H4.75Z"
+      fill="currentcolor"
+    />
+  </g>
+  <defs>
+    <clipPath id="box">
+      <rect width="16" height="16" fill="white" transform="translate(4 4)" />
+    </clipPath>
+  </defs>
```

Comment:
I just had to use this icon in the website and adjusted it to not need all this extra junk

```
<path
  d="M14.25 4H13.5V5.5H14.25H17.4406L10.4688 12.4688L9.9375 13L11 14.0594L11.5312 13.5281L18.5 6.55938V9.75V10.5H20V9.75V4.75V4H19.25H14.25ZM4.75 5H4V5.75V19.25V20H4.75H18.25H19V19.25V13.75V13H17.5V13.75V18.5H5.5V6.5H10.25H11V5H10.25H4.75Z"
  fill="currentcolor"
/>
```

Reply:
🤌